helper_funcs.inc.php

```diff
@@ -210,6 +210,11 @@ function pwg_privacy_verify_access ($img_id, $req_path) {
 
 	//file is original uploaded image
 	if ( strpos($req_path, $path) === 0 ) {
+		//file is a video, serve it.
+		if ( strpos($path, '.mp4') !== 0 ) {
```

Comment:
Notice that piwigo provides a `get_extension` method (used couple of times in this file).
So you can probably just do `get_extension($path) === 'mp4'`.
With multiple valid extensions you'll probably need to use `in_array` to see if the extension is a valid one